packages/core/src/types.ts

```diff
@@ -113,3 +113,21 @@ export type Provider =
   | 'strava'
   | 'gitlab'
   | 'bitbucket'
+
+// TODO share with hasura-auth
+export interface JWTHasuraClaims {
+  // ? does not work as expected: if the key does not start with `x-hasura-`, then it is typed as `any`
+  // [claim: `x-hasura-${string}`]: string | string[]
+  [claim: string]: string | string[] | null
+  'x-hasura-allowed-roles': string[]
+  'x-hasura-default-role': string
+  'x-hasura-user-id': string
+  'x-hasura-user-is-anonymous': string
+}
+
+// https://hasura.io/docs/1.0/graphql/core/auth/authentication/jwt.html#the-spec
```

Comment:
Could you please change this to a proper JSDoc? Our DocGen would pick up this comment and add this to the markdown output.

Reply:
Apparently the DocGen didn't see this change because `.lintstagedrc.json` doesn't contain this package. Could you please add it to the regex?

Reply:
done